#! /bin/sh /usr/share/dpatch/dpatch-run
## 01collstatus.dpatch by  <az@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix for #625645, to allow collection status without key 

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' duplicity-0.6.13~/duplicity duplicity-0.6.13/duplicity
--- duplicity-0.6.13~/duplicity	2011-05-21 17:44:05.801677403 +1000
+++ duplicity-0.6.13/duplicity	2011-05-21 17:44:32.558815239 +1000
@@ -1141,7 +1141,8 @@
     check_resources(action)
 
     # check archive synch with remote, fix if needed
-    sync_archive()
+    if action not in ["collection-status"]:
+        sync_archive()
 
     # get current collection status
     col_stats = collections.CollectionsStatus(globals.backend,
